### PR TITLE
12.0 mig hr contract multi job

### PR DESCRIPTION
**Pull request by eslAmer**

### CODE REVIEW

File: hr_contract_multi_job/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (tarteo) [inline hr_contract_multi_job/README.rst:1]: Can you use the split readme like in the template: https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

File: hr_contract_multi_job/models/hr_contract_job.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Copyright (C) 2014 Savoir-faire Linux. All Rights Reserved.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import models, fields
+from odoo import models, fields, api
```
Comment (tarteo) [inline hr_contract_multi_job/models/hr_contract_job.py:4]: api is not needed here it's not used.

**Review by tarteo — CHANGES_REQUESTED** [review]
Looks good just some small comments.

**Comment by pedrobaeza** [comment]
There's already a migration of this module in #482

**Pull request closed without merging**